src/git/install.sh

```diff
@@ -61,40 +93,70 @@ receive_gpg_keys() {
     fi
 }
 
-apt_get_update()
-{
-    if [ "$(find /var/lib/apt/lists/* | wc -l)" = "0" ]; then
-        echo "Running apt-get update..."
-        apt-get update -y
+pkg_mgr_update() {
+    if type apt >/dev/null 2>&1; then
```

Comment:
Should this be, same as your check inside `check_packages()`:
```suggestion
    if [ ${INSTALL_CMD} = "apt-get" ]; then
```

Reply:
Done.